app/models/order.rb

```diff
@@ -497,4 +507,24 @@ def order
   def offer
     nil
   end
+
+  def validate_shipment_date
+    is_valid = shipment_date >= Date.current
```

Comment:
This will mean we can never update shipment that have been sent in the past. I think it may be best to remove this validation entirely or just leave it on 'create' and not 'update'.

Reply:
Sure will just leave it on create